src/services/PermissionService.js

```diff
@@ -0,0 +1,53 @@
+import { prepareRequest } from 'boot/axios';
+
+/**
+ * Generates permission keys based on the provided array.
+ * @param {Array} permissions - Permissions array.
+ * @param {string} permissions.entity - The entity for which the permission is granted.
+ * @param {string} permissions.action - The action associated with the permission.
+ * @param {string} permissions.libraryId - Optional library ID associated with the permission.
+ * @returns {Array} An array of permission objects with added 'key' property.
+ */
+export function generatePermissionKeys(permissions) {
```

Comment:
generate only one permission key, make this methode return string

Reply:
done